README.md

```diff
@@ -77,14 +77,79 @@ If the destination Mimir installation is part of a Grafana cloud instance the `-
   -write-endpoint https://_username_:_password_@_grafana_net_instance_/api/v1/push
 where the exact server details can be found on Prometheus instance details page for the stack on grafana.com
 
-The _username_ is the numeric `Username / Instance ID`
-The _password_ is the Grafana Cloud API Key with `metrics push` privileges/role.
-The _grafana_net_instance_ is server part of the URL to push Prometheus metrics.
+The `_username_` is the numeric `Username / Instance ID`
+The `_password_` is the Grafana Cloud API Key with `metrics push` privileges/role.
```

Comment:
Should we mention the recommended role is `MetricsPublisher`? As it only has permission to do writes

Reply:
Updated it with the correct role name.